invesalius/net/pedal_connection.py

```diff
@@ -40,32 +40,32 @@ def __init__(self):
 
         self._midi_in = None
         self._active_inputs = None
-        self._callbacks = {}
+        self._callback_infos = []
```

Comment:
I assume that the dict error that we were having is now fixed by using a list of dicts, is that correct?